### DISCUSSION TITLE
Create discussion threads for each release and prerelease

### DISCUSSION THREAD

**Discussion by ssbarnea** [release-drafter/release-drafter#1209] [General]
I seen that recently there was beta pre-release but it contains no changelist and there is no discussion thread created with it.

I would be awesome to create threads for each release and pre-release, so we can give feedback regarding them directly on these. That is just one checkbox in release form you need to click!

I am also curious what is baking in 6.0... and eager to provide early feedback.

**Comment by jetersen** [comment]
The discussion is the PR and issues linked, the PR has list of tasks (although not complete and is ever expanding, I want make sure we get a good major release):
https://github.com/release-drafter/release-drafter/pull/1204

Several reason exist for no changelist is that is currently early development and I wanted to get feedback on the CLI.
Reason for the release happening on a side branch and not in master branch is the fact that I know repos are using release-drafter master branch for GitHub actions and it is not feature complete, it is basically a rewrite of the codebase.

Once we are more feature complete and moved tests or written new tests for v6 we can start merging it and do more prereleases (I want to use release-drafter to iron out the prerelease experience).
I was hoping to include conventional commit support as well for v6.

Basically I want to close as many issues as possible with v6 and make it equal if not superior to any releasing tool that exist.

I released a early version for the CLI to be tested in https://github.com/release-drafter/release-drafter/issues/1146

You can provide feedback in the PR or issue or here. I am reading all of it.

However I am but a one man band on the project.

**Reply by jetersen** [reply]
One issue that exist is we still do not have support release-drafter to create an discussion for a release.
